ksql-rest-model/src/main/java/io/confluent/ksql/rest/entity/HostStoreLags.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2020 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"; you may not use
+ * this file except in compliance with the License. You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.rest.entity;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static java.util.Objects.requireNonNull;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableMap;
+import com.google.errorprone.annotations.Immutable;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * Represents a host's lag information, and when the lag information was collected.
+ */
+@Immutable
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class HostStoreLags {
+
+  private final ImmutableMap<QueryStateStoreId, StateStoreLags> stateStoreLags;
+  private final long updateTimeMs;
+
+  @JsonCreator
+  public HostStoreLags(
+      @JsonProperty("stateStoreLags") final Map<QueryStateStoreId, StateStoreLags> stateStoreLags,
+      @JsonProperty("updateTimeMs") final long updateTimeMs) {
+    this.stateStoreLags = ImmutableMap.copyOf(requireNonNull(stateStoreLags, "stateStoreLags"));
+    this.updateTimeMs = updateTimeMs;
+  }
+
+  public StateStoreLags getStateStoreLags(final QueryStateStoreId queryStateStoreId) {
+    return stateStoreLags.get(queryStateStoreId);
```

Comment:
As with the other `get` method - avoid returning `null`s.

Reply:
I changed it to return Optional